examples/mautic-example-nginx-ssl-managedDB/README.md

```diff
@@ -0,0 +1,22 @@
+This example shows how to configure nginx to act as a reverse proxy with ssl support for mautic container.
+
+#### What is handled in this docker-compose.yml:
+
+* Setup of mysql and mautic containers
+* Setup of mysql db on managed or remote server
+* Setup of a nginx container with custom vhost configuration (nginx.conf)and secure with letsencrypt certs
+
+#### How to use this example:
+
+1. run ```docker-compose up``` in this directory
+2. add these lines to your /etc/hosts file 
+```127.0.0.1       pilotdev9.eastbayinfo.org ```
+``` 127.0.4.123 mautic.local ```
+3. access https://pilotdev9.eastbayinfo.org
+4. go through Mautic setup (fill ```mauticdbpass``` as mysql password on db setup page
+6. test Mautic
+
+#### Notes
+* The certificate should be made trustworthy. It may not be enough to just click trough the warning in some browsers.
```

Comment:
Can you add details on how to do this? And/or link out to a reputable resource that explains to the user how to do it?

```suggestion
* The certificate should be made trustworthy. It may not be enough to just click through the warning in some browsers.
```